cockatrice/src/player.cpp

```diff
@@ -1313,8 +1313,8 @@ void Player::actMoveTopCardsUntil()
     QString expr = previousMovingCardsUntilExpr;
     for (;;) {
         bool ok;
-        expr =
-            QInputDialog::getText(game, "Take top cards until", "Select card (accepts search syntax)", {}, expr, &ok);
+        expr = QInputDialog::getText(game, "Put top cards on stack until", "Card name (accepts search syntax)", {},
```

Comment:
the box is much more useful for stuff that isn't the name of the card, eg mv<4 -t:land for cascading, I'd prefer to avoid the use of "name"

Reply:
we might consider changing the text in the deck editor as well, it currently says "search by card name" before typing

Reply:
> the box is much more useful for stuff that isn't the name of the card, eg mv<4 -t:land for cascading, I'd prefer to avoid the use of "name"

Nice, I had no clue! And the dialog is not helping here, see my question in the opening post: 
"Which search syntax is actually supported in the follow-up dialog?"
I did only try with a placeholder "goblin *", and it did not work.

Reply:
I went with `Card name (or advanced syntax)` now and used the same phrasing in the deck editor search. 👍 
That way people hopefully get the link that it's the same search syntax.

I still think the advanced search help is a bit hidden though. E.g. a colored icon over a grayed out one would signal that it can be interacted with, and a hover label on the icon would be great to explain what it's about.

Reply:
I don't like the "advanced" label, we only have one search syntax, https://scryfall.com/docs/syntax doesn't call it advanced either, maybe just call them "search expressions" instead of syntax which might be a less used word.

Reply:
I like this suggestion!